firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FeedbackSender.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.appdistribution.impl;
+
+import android.graphics.Bitmap;
+import androidx.annotation.NonNull;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.appdistribution.FirebaseAppDistribution;
+import com.google.firebase.appdistribution.internal.FirebaseAppDistributionProxy;
+
+/** Sends tester feedback to the Tester API. */
+class FeedbackSender {
+
+  private final FirebaseAppDistributionTesterApiClient testerApiClient;
+
+  FeedbackSender(FirebaseAppDistributionTesterApiClient testerApiClient) {
+    this.testerApiClient = testerApiClient;
+  }
+
+  @NonNull
```

Comment:
I noticed you removed this annotation in UpdateProgress.  How do you decide it's needed here?

Reply:
It wasn't needed in UpdateProgress because those return types (long) were not nullable.

Reply:
Actually I don't think this is necessary either. In this case it's because this is not a publicly exposed API. I'll remove it.